app/models/oclc/bib_processing_file.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+module Oclc
+  class BibProcessingFile
```

Comment:
I was wondering if you have strong opinions on the OCLC job naming. I think the "exception" jobs and the "processing" jobs should have parallel names, and I went with DataSync because that's the Alma / OCLC process that these particular jobs relate to. I also think we do a lot of kinds of "BibProcessing", so in the future that name could be confusing. 

Would you be open to re-naming the classes to `DataSyncProcessingFile` and `DataSyncProcessingJob`?